src/ray/object_manager/common.cc

```diff
@@ -168,7 +168,7 @@ Status PlasmaObjectHeader::ReadAcquire(
     const std::unique_ptr<std::chrono::steady_clock::time_point> &timeout_point) {
   RAY_CHECK(sem.header_sem);
 
-  RAY_RETURN_NOT_OK(TryToAcquireSemaphore(sem.header_sem, timeout_point));
+  RAY_RETURN_NOT_OK(TryToAcquireSemaphore(sem.header_sem));
```

Comment:
Could you add a comment why timeout is not applied here in the code, same for the other place